internal/blocktx/health_check_test.go

```diff
@@ -79,14 +79,19 @@ func TestCheck(t *testing.T) {
 					},
 				}}
 			}}
-			server := blocktx.NewServer(storeMock, logger, pm, 0)
-			resp, err := server.Check(context.Background(), req)
-			require.NoError(t, err)
+			sut := blocktx.NewServer(storeMock, logger, pm, 0)
+
+			// when
+			resp, err := sut.Check(context.Background(), req)
 
+			// then
+			require.NoError(t, err)
 			require.Equal(t, tc.expectedStatus, resp.Status)
 
+			// cleanup
```

Comment:
You can remove `processor` from these test - it is not used

Reply:
Removed